src/shared/ui-kit/components/TermsConditionModal/TermsConditionModal.tsx

```diff
@@ -0,0 +1,178 @@
+import { Button, Flex, Modal, ModalBody, ModalContent, ModalHeader, ModalOverlay } from "@chakra-ui/react";
+import { Color, Text } from "@shared/ui-kit";
+import { LocalStorageKeys } from "../constants";
+import { useState } from "react";
+
+export function TermsConditionModal() {
+  const localStorageState = localStorage.getItem(LocalStorageKeys.ACCEPTED_TERMS_AND_CONDITIONS);
+  const [acceptedCondition, setAcceptedCondition] = useState(localStorageState ? JSON.parse(localStorageState) : false);
+
+  const onAccept = () => {
+    setAcceptedCondition(true);
+    localStorage.setItem(LocalStorageKeys.ACCEPTED_TERMS_AND_CONDITIONS, JSON.stringify(true));
+  };
+
+  if (acceptedCondition) {
+    return <></>;
+  }
+  return (
+    <Modal
+      variant="termsConditionModal"
+      isCentered
+      isOpen={true}
+      onClose={() => {
+        console.log("[ANS] close modal");
```

Comment:
can we remove this console if not required.?

Reply:
Done.